dataland-frontend/tests/e2e/specs/dev/DataIntegrity.ts

```diff
@@ -1,9 +1,11 @@
+import {visitAndCheckAppMount} from "../../support/commands";
+
 describe("EU Taxonomy Data", () => {
   it("Check for a dataset, that bears decent data, that the data is present and displayed", () => {
     cy.restoreLoginSession();
     cy.retrieveDataIdsList().then((dataIdList: Array<string>) => {
       cy.intercept("**/api/data/eutaxonomies/*").as("retrieveTaxonomyData");
-      cy.visit("/data/eutaxonomies/" + dataIdList[0]);
+      visitAndCheckAppMount("/data/eutaxonomies/" + dataIdList[0]);
```

Comment:
Generell:  Man könnte auch `cy.visitAndCheckAppMount` schreiben.
Mir ist aufgefallen, dass wir das grundsätzlich bei unseren cypress-Funktionen manchmal machen (z.B. bei `cy.restoreLoginSession`, und manchmal nicht.
Sollen wir hier konsistent bleiben?   Wenn wir es direkt hinschreiben, brauchen wir die Funktionen in unserer `commands.ts` auch nicht über
`Cypress.Commands.add("visitAndCheckAppMount", visitAndCheckAppMount);`
als Cypress commands registrieren.